packages/direflow-component/src/services/materialUiStylesHandler.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { IDireflowPlugin } from '../types/DireflowConfig';
+
+const handleMaterialUiStyle = (app: React.FC<any> | React.ComponentClass<any, any>, plugins: IDireflowPlugin[] | undefined)
+    : { mountPoint?: Element, app: React.FC<any> | React.ComponentClass<any, any> } => {
+
+    if (plugins?.find((plugin) => plugin.name === 'material-ui')) {
+        try {
+            const jssLib = require('jss');
+            const materialUiStyles = require('@material-ui/core/styles');
+            const { create } = jssLib;
+            const { jssPreset } = materialUiStyles;
+            const { StylesProvider } = materialUiStyles;
+
+            const mountPoint = document.createElement('span');
+
+            const jss = create({
+                ...jssPreset(),
+                insertionPoint: mountPoint
+            });
+
+            return { mountPoint, app: (<StylesProvider jss={jss}>{app}</StylesProvider>) };
```

Comment:
I'm not able to build due to typing related problems here.
`Type 'Element' is not assignable to type 'FC<any> | ComponentClass<any, any>'.
  Type 'Element' is not assignable to type 'ComponentClass<any, any>'.
    Type 'Element' provides no match for the signature 'new (props: any, context?: any): Component<any, any, any>'.`